hadoop/src/main/java/org/carbondata/hadoop/util/ObjectSerializationUtil.java

```
@@ -33,18 +37,29 @@ public static String convertObjectToString(Object obj) throws IOException {
       oos = new ObjectOutputStream(gos);
       oos.writeObject(obj);
     } finally {
-      oos.close();
-      gos.close();
-      baos.close();
+      try {
+        if (oos != null) {
+          oos.close();
+        }
+        if (gos != null) {
+          gos.close();
+        }
+        if (baos != null) {
+          baos.close();
+        }
+      } catch (IOException e) {
+        LOG.error(e);
```

should throw the exception again, right?